lib/workflows/hv_service_actions.py

```diff
@@ -0,0 +1,42 @@
+from openstack.connection import Connection
+from openstack_api.openstack_service import disable_service, enable_service
+
+
+def hv_service_disable(
+    conn: Connection,
+    hypervisor_name: str,
+    service_binary: str,
+    disabled_reason: str,
+) -> None:
+    """
+    Disables an Openstack service
+    :param conn: Openstack connection
+    :param hypervisor_name: (str): The name or ID of the hypervisor.
```

Comment:
See other comments about typing
```suggestion
    :param hypervisor_name: The name or ID of the hypervisor.
```